packages/lit-element/src/lit-element.ts

```diff
@@ -57,6 +57,26 @@ export const UpdatingElement = ReactiveElement;
 
 const DEV_MODE = true;
 
+let issueWarning: (warning: string) => void;
+
+if (DEV_MODE) {
+  // Ensure warnings are issued only 1x, even if multiple versions of Lit
+  // are loaded.
+  const issuedWarnings: Set<string | undefined> =
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (globalThis as any)['litIssuedWarnings'] ??
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    ((globalThis as any)['litIssuedWarnings'] = new Set());
```

Comment:
If we added appropriate types, a lot of this could would be much more readable (eliminate the casts and the disable any):
```ts
declare global {
  var litIssuedWarnings: string[];
  var litElementVersions: string[];
}
```